schemas/IDLs/org.gel.models.report.avro/6.0.0/CommonInterpreted.avdl

```diff
@@ -6,6 +6,9 @@ protocol CommonInterpreted {
 
     import idl "CommonParticipant.avdl";
 
+
+enum TimeUnit {years, months, weeks, days, hours, minutes, na}
```

Comment:
This is only used in the `AgeRange` object, can we simplify it by indicating in the comments that age is defined in years? This wouold allows us to get rid of the whole `TimeUnit` enum.

Reply:
you are missing seconds :p